internal/cloud/e2e/helper_test.go

```diff
@@ -213,7 +229,11 @@ findTfVersion:
 			t.Fatalf("Could not retrieve list of terraform versions: %v", err)
 		}
 		for _, item := range tfVersionList.Items {
-			if item.Version == version {
+			desiredVersion, err := goversion.NewVersion(item.Version)
+			if err != nil {
+				t.Fatalf(fmt.Sprintf("Error instantiating go-version for %s", item.Version))
+			}
+			if desiredVersion.Core().GreaterThanOrEqual(baseVersion.Core()) {
```

Comment:
Regarding updating the func doc, i'll definitely do that. The entire point of this is to make sure that w/e version exists remotely, it has >= 1.1 so that it has the `cloud` block available.

That was the reason for making this a `>=` rather than an `==`. The only concern of this function is so that ultimately we don't get an error saying `"What is this cloud block you hav here?"`. And thats why 1.2, 1.3 don't matter as much. If we ever get to a point where we are writing e2e tests against particular terraform versions, than I can see an `==` making more sense.